web/static/js/components/Achievement.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Request from '../Request'
+import Icon from './Icon'
+
+class Achievement extends React.Component {
+
+  isCompleted() {
+    const { value, threshold } = this.props.achievement
+    return value >= threshold
+  }
+
+  classNames() {
+    const names = ['Achievement']
+    if (this.isCompleted()) {
+      names.push('Achievement--completed')
+    }
+    return names.join(' ')
+  }
+
+  render() {
+    const { achievement } = this.props;
+    return (
+      <li className={this.classNames()}>
+        <Icon className="Achievement-icon" name="achievement" />
+        <div className="Achievement-text">
+          <div><strong>{achievement.name}</strong></div>
+          <div>{achievement.description}</div>
+          {!this.isCompleted() &&
+            <div>{Math.floor(achievement.value / achievement.threshold * 100)}% Complete</div>
+          }
```

Comment:
Another option for showing progress:

<img width="234" alt="screen shot 2017-02-26 at 5 51 44 am" src="https://cloud.githubusercontent.com/assets/664341/23339050/ba58bc50-fbe7-11e6-9423-6c40b16c74fe.png">